app/screens/wallet/Backup.js

```diff
@@ -0,0 +1,186 @@
+`;
+
+const BoldBackupOptiosHeader = styled(BaseText)`
+  font-weight: bold;
+  margin-bottom: 24px;
+`;
+
+const GrayText = styled(BaseText)`
+  color: ${smColors.darkGray};
+`;
+
+const Separator = styled.div`
+  border-bottom: 1px solid ${smColors.borderGray};
+  margin-bottom: 24px;
+`;
+
+const BackupBoxesWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-bottom: 24px;
+`;
+
+// $FlowStyledIssue
+const ActionLink = styled(BaseText)`
+  user-select: none;
+  color: ${({ color }) => color};
+  cursor: pointer;
+  &:hover {
+    opacity: 0.8;
+  }
+  &:active {
+    opacity: 0.6;
+  }
+`;
+
+// $FlowStyledIssue
+const BackupBox = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  border: 1px solid ${({ borderColor }) => borderColor};
+  flex: 1;
+  &:first-child {
+    margin-right: 16px;
+  }
+  border-radius: 2px;
+  padding: 24px 18px;
+`;
+
+const BackupTopWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  margin-bottom: 40px;
+`;
+
+const SecurityLogo = styled.img`
+  height: 66px;
+  width: 54px;
+  margin-right: 30px;
+`;
+
+const BackupHeader = styled.span`
+  font-size: 24px;
+  font-weight: bold;
+  line-height: 33px;
+  color: ${smColors.darkGray};
+`;
+
+const BackupTitleWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+`;
+
+const Recommended = styled.div`
+  align-self: flex-end;
+  padding-left: 24px;
+`;
+
+const bodyText = {
+  file: 'Take your wallet file and save a copy of it in a secure location - We recommend a USB key or a external drive.',
+  mnemonic: 'A list of words you keep on a physical paper or in your password manager, that can be used to recover your wallet on any device.'
+};
+const noteText = {
+  file: 'NOTE: You will still need your passphrase to restore the wallet from the file.',
+  mnemonic: 'NOTE: Works even if you lost your passphrase or wallet backup file'
+};
+
+type Props = {};
+
+class Backup extends Component<Props> {
+  render() {
+    return (
+      <Wrapper>
+        <Header>Wallet Backup</Header>
+        <SubHeader>Your wallet file is encrypted with your passphrase on your computer, but We recommend that you’ll backup your wallet for additional security.</SubHeader>
+        <Separator />
+        <BoldBackupOptiosHeader>How would you like to backup your wallet?</BoldBackupOptiosHeader>
+        <BackupBoxesWrapper>
+          {this.renderBackupBox('file')}
+          {this.renderBackupBox('mnemonic')}
+        </BackupBoxesWrapper>
+        <ActionLink onClick={this.learnMoreAboutSecurity} color={smColors.green}>
+          Learn more about wallet security and backup
+        </ActionLink>
+      </Wrapper>
+    );
+  }
+
+  renderBackupBox = (mode: 'file' | 'mnemonic') => {
+    const isFileMode = mode === 'file';
+    const shieldLogo = isFileMode ? shieldIconGreenOne : shieldIconOrangeTwo;
```

Comment:
shieldLogo, securityLevel, backupMode, linkText - all of these are constants and don't depend on any external source of truth. Extract them to constant outside of class and unite with bodyText/noteText

Reply:
ok